webpack/components/extensions/HostDetails/Tabs/TraceTab.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useCallback } from 'react';
```

Comment:
File should probably be named `TracesTab.js` (not `TraceTab.js`) to match the name of the component

Reply:
Fixed.